letsencrypt/errors.py

```diff
@@ -24,6 +24,8 @@ class SubprocessError(Error):
 class CertStorageError(Error):
     """Generic `.CertStorage` error."""
 
+class ConfiguratorError(Error):
```

Comment:
I think the name is confusing - it suggests errors raised by any configurator (plugin that is both an authenticator and installer). Maybe `PluginSelectionOrSetupError`? ;)

Documentation below ("plugin/configurator") should just say "configurator", according to the definition of word "configurator" that is used throughout the code base.